samples/templates/express-embedded-auth-with-sdk/web-server/views/enroll-poll.mustache

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    {{>head}}
+    {{#pollForResult.refresh}}
+    <script type="text/javascript">
+      setInterval(function () {
```

Comment:
`pollForResult.refresh` is not guaranteed to be the same value each time a poll request returns, the server implements back-off strategies (which may increase the refresh value overtime). I would change this to a `setTimeout` rather than a `setInterval`

Reply:
In this case `setInterval` is a kind of a safety net for the case when polling form is not available in DOM at script run time. If the form is available, it will be submitted(page reloaded) and `setInterval` callback won't fire anymore.